src/fairseq2/models/wav2vec2/model.py

```diff
@@ -113,10 +113,15 @@ def __init__(
         self.logit_temp = logit_temp
         self.diversity_loss_weight = diversity_loss_weight
 
-    def forward(self, batch: SequenceBatch) -> Wav2Vec2Output:
-        """
+    def extract_features(self, batch: SequenceBatch) -> Wav2Vec2Features:
+        """Extract features from the input sequences.
         :param batch:
             The batch of sequences to process.
+
+        :returns:
+            - A `Wav2Vec2Features` object, consisting of the encoder output, targets
```

Comment:
Again, very nit. `- ` causes a bullet point in sphinx. You can start the sentence with "A `Wav2Vec2Features` object, .."

Reply:
Ah, was matching how the `run_frontend` formatting was, sorry.